_posts/2018-01-02-this-is-not-a-test.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title:  "This Is Not a Test"
+date:   2018-01-02 12:00:00 -0400
+categories: culture interviews
+author: Janice Smith
+---
+
+At Forward Financing, we are always trying to improve how we work.
+We have some core tenets to how we work as a team. Our director of engineering,
+Patrick Hereford, wrote a great post about that recently. You should check it out [here](http://tech.forwardfinancing.com/engineering/culture/2017/09/30/our-engineering-culture-methodologies-and-philosophies.html){:target='_blank'}
+
+We have been in the middle of growing the engineering team numbers here,
+in a push that will eventually more than double the size of our group.
+This growth is key to support all of the new work coming our way,
+that marks an increased investment in tech here at Forward Financing.
+These positions are for different levels of engineers and for both local and remote team members.
+This is a very exciting time for us, but how successful we are may ultimately depend on who we hire.
+
+We care about finding team members interested in joining a blameless culture rooted in learning.
+We do this mainly with code review and pairing.
+There are lots of great engineers out there, all with their own styles and preferred ways of working in a team.
+We really wanted to focus on finding team members who would strengthen the values we have in place and bring in different points of view.
+
+We talked a lot about how we might tailor an interview solution that fits our team, and we came together on pair programming. We do not practice strict formal pairing here as a whole,
+but we really like to collaborate, especially on large features.
+It is common for each person to either hop on a call with another engineer about a challenge, an idea, or to talk over code review. Settling on that as the common factor, being able to communicate well with any new person was really key.
```

Comment:
~Settling on that as the common factor,~ being able to communicate well with any new person was ~really~ key.

I'm seeing a lot of "really"s in here.